### ISSUE TITLE
Cannot use pyhecdss with Numpy >= 2

### ISSUE THREAD

**Issue by bzindovic** [CADWRDeltaModeling/pyhecdss#35]
* pyhecdss version: 1.4.0+2.g9d14b63
* Python version: 3.11
* Operating System: Windows 10 64-bit

### Description

For now, it seems that pyhecdss works with numpy < 2 and I have to downgrade numpy to 1.26.4 (it seems to be the last working for me). As Numpy version 2 introduces some breaking changes, are there any plan to support it?

### What I Did

I did a simple import of the package:
```
import pyhecdss
```

and it fails immediately with the following output:
```
A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.
```

**Comment by kjnam** [comment]
We are aware of the issue, and we will work on it soon. Thanks for reporting the issue.

**Comment by dwr-psandhu** [comment]
Till then the workaround is to pin python<3.12 and numpy<2

**Comment by dwr-psandhu** [comment]
Fixed by #36 and #31. Now only supporting python 3.11 and numpy > 2 for future updates. Older versions will still work but users will have to pin numpy < 2 for those versions.
